src/pages/experiments/[experimentId]/plots-and-tables/frequency/index.jsx

```diff
@@ -167,7 +167,7 @@ const FrequencyPlotPage = ({ experimentId }) => {
   };
 
   const renderCSVbutton = () => (
-    <ExportAsCSV data={csvData} filename={csvFilename} disabled={false} />
+    <ExportAsCSV data={csvData} filename={csvFilename} />
```

Comment:
Is there any condition where the export as csv button might be disabled?

Reply:
yeah in the volcano plot